packages/core/src/js/feedback/FeedbackForm.styles.ts

```diff
@@ -0,0 +1,54 @@
+import type { FeedbackFormStyles } from './FeedbackForm.types';
+
+const defaultStyles: FeedbackFormStyles = {
```

Comment:
Let's match the default with JS.

Mainly I've noticed submitButton backgroundColor (JS has `rgba(88, 74, 192, 1)`) and test color `#2b2233`

Reply:
Tried to [align with the light theme of JS](https://github.com/getsentry/sentry-javascript/blob/a5fc322236211551f2c22a5824e29612d224cd94/packages/feedback/src/core/createMainStyles.ts#L11) with 30a7b1093eed42d56e410e414d58a1e7d22a4db3

<details><summary>Screenshot</summary>
<p>

![Simulator Screenshot - iPhone SE (3rd generation) - 2024-12-13 at 19 40 33](https://github.com/user-attachments/assets/95dc1ab3-74e4-4206-a28a-d671314d1175)

</p>
</details>

Reply:
Okay, thanks you. 

Somehow on the screenshot the color looks like different shade, but if the color code is the same as web, it's good.